static/js/containers/ProfileFormContainer.js

```diff
@@ -47,8 +47,10 @@ class ProfileFormContainer extends React.Component {
 
   fetchProfile = () => {
     const { dispatch, profiles, params: { username } } = this.props;
-    if (profiles[username] === undefined || profiles[username].getStatus === undefined) {
-      dispatch(fetchUserProfile(username));
+    let _username = username || SETTINGS.username;
```

Comment:
I think that would make more sense, as is were just increasing the degree to which it's coupled to particular use-cases.